sssd_test_framework/utils/sssctl.py

```diff
@@ -90,3 +93,28 @@ def passkey_register(
             )
 
         return result.stdout_lines[-1].strip()
+
+    def cache_expire(self, *args: str) -> SSHProcessResult:
+        """
+        Run ``sssctl cache-expire`` command.
+
+        :param args: Additional arguments.
+        :type args: str
+        """
+        return self.host.ssh.exec(["sssctl", "cache-expire", *args])
+
+    def cache_ls(self) -> SSHProcessResult:
+        """
+        Run ``ls 'cache_path'`` command.
+        Result output of ``ls`` is stored in result.stdout
+        """
+        return self.host.ssh.exec(["ls", self.cache_path])
+
+    def remove_cache_files(self) -> SSHProcessResult:
```

Comment:
Why is this needed?

This is not part of sssctl command so it should not be here.

If anything, there is `cache-remove`, but I'm not sure from the top of my head if it removes memory cache.